### PR TITLE
Added negative button in Dialog

### PR DESCRIPTION
**Pull request by SanjaySargam**
## Purpose / Description
It's good to have a negative button in the dialog

## Fixes
Related #13724

## Approach
Added negative button

## How Has This Been Tested?

Tested on Emulator (Pixel 5 API 29)

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/utils/ImportUtils.kt [hunk 1]
```diff
@@ -299,6 +298,7 @@ object ImportUtils {
                         AnkiActivity.finishActivityWithFade(activity)
                     }
                 }
+                negativeButton(R.string.dialog_cancel)
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/utils/ImportUtils.kt:301]: This should be removed. This dialog is a dialog notifying the user of an error and doesn't need a cancel button. Adding this cancel button will also create a bug.

**Comment by BrayanDSO** [comment]
Do you have screenshots of the dialogs you've changed? Because it doesn't make sense to add a negative button to some of them, since they are only warning that an error has ocurred

File: AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.kt [hunk 2]
```diff
@@ -1819,6 +1819,7 @@ class NoteEditor : AnkiActivity(), DeckSelectionListener, SubtitleListener, Tags
                 val et2 = v.findViewById<EditText>(R.id.note_editor_toolbar_after)
                 addToolbarButton(etIcon.text.toString(), et.text.toString(), et2.text.toString())
             }
+            negativeButton(R.string.dialog_cancel)
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.kt:1822]: This is not needed, there's already a cancel button(see the toolbarDialog property).

File: AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.kt [hunk 3]
```diff
@@ -1841,6 +1842,7 @@ class NoteEditor : AnkiActivity(), DeckSelectionListener, SubtitleListener, Tags
                     currentButton
                 )
             }
+            .negativeButton(R.string.dialog_cancel)
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.kt:1845]: This is not needed, there's already a cancel button(see the toolbarDialog property).

**Review by lukstbit — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/ui/AppCompatPreferenceActivity.kt [hunk 4]
```diff
@@ -323,6 +323,7 @@ abstract class AppCompatPreferenceActivity<PreferenceHack : AppCompatPreferenceA
                 positiveButton(R.string.dialog_ok) { dialogInterface ->
                     dialogInterface.dismiss()
                 }
+                negativeButton(R.string.dialog_cancel)
```
Comment (SanjaySargam) [inline AnkiDroid/src/main/java/com/ichi2/ui/AppCompatPreferenceActivity.kt:326]: I realised that this is also not needed as it is showing an exception that error is occurred

So, I'm closing this PR

**Pull request closed without merging**